### PR TITLE
chore: Add metadata cmd unit test.

### PR DESCRIPTION
**Pull request by qz267**
This is a follow up of https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/pull/2660.

To add coverage for the function implementations in cmd.go file.

Initialized two test cases:
- TestCmdExecution(): using `cobra.ExecuteC()` to simulate command execution without actually running, enabled adding more cases to cover the subcommands in the future. 
- TestCreateBlueprintDisplayMetadata(): cover the scenarios @tjy9206 found in the previous PR.

### CODE REVIEW

**Review by tjy9206 — APPROVED** [review]

File: cli/bpmetadata/cmd_test.go [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+func TestCreateBlueprintDisplayMetadata(t *testing.T) {
+	tests := []struct {
+		description string
+		bpPath      string
+		bpDisp      *BlueprintMetadata
+		bpCore      *BlueprintMetadata
+		expectErr   bool
+	}{
+		{
+			description: "create metadata with nil Spec.UI.Input",
```
Comment (bharathkkb) [inline cli/bpmetadata/cmd_test.go:56]:
```suggestion
			description: "create metadata with nil display metadata Spec.UI.Input",
```

**Review by bharathkkb — APPROVED** [review]
Thanks for the followup!

File: cli/bpmetadata/cmd_test.go (hunk 1 above)
Comment (qz267) [reply cli/bpmetadata/cmd_test.go]: Done

File: cli/bpmetadata/cmd_test.go [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.description, func(t *testing.T) {
+			cmd := &cobra.Command{Use: "cft"}
+			cmd.SetArgs(tt.args)
+
+			execCmd := func() error {
+				_, err := cmd.ExecuteC()
+				return err
```
[the remark this answers is not in the document]
Comment (qz267) [reply cli/bpmetadata/cmd_test.go]: Done

**Pull request merged**